notebooks/407-person-tracking-webcam/preprocess.py

```diff
@@ -0,0 +1,474 @@
+import numpy as np
```

Comment:
If this block is only designed for Paddle model, can we put this file into ```ocsor_utils``` fold ?